### ISSUE TITLE
Markdown Editor overlay bug

### ISSUE THREAD

**Issue by James-Jefferies** [laravel-filament/filament#1540]
### Package

filament/filament

### Package Version

V2.10.4

### Laravel Version

v8.83.0

### Livewire Version

V2.10.0

### PHP Version

PHP 8.1.2

### Bug description

When using a standard none edit markdown editor it appears to show an overlay preview of anything typed

![image](https://user-images.githubusercontent.com/98420603/153388768-13a3fa95-232b-4b7f-b454-92e6c491f8a6.png)

### Steps to reproduce

```
public static function form(Form $form): Form
    {
        return $form
            ->schema([
                MarkdownEditor::make('content'),
                                // ...
                            ]);
    }
```

**Comment by ryangjchandler** [comment]
@danharrin Is this maybe a dark mode bug?

**Comment by danharrin** [comment]
Don't think so. I can't replicate this. What browser are you using @James-Jefferies?

<img width="745" alt="Screenshot 2022-02-10 at 11 42 43" src="https://user-images.githubusercontent.com/41773797/153402133-20d01a4f-b6b5-4205-bbb7-6352e1eef50b.png">

**Comment by James-Jefferies** [comment]
Ah interesting, I use Firefox, but I seem to be able to replicate this in both chrome and FF. I had a look using alpine tools and it seems to be showing the state as the overlay for some reason, I manually set it to test 3 in Alpine tools to determine which element was showing. 

I have cleared all views and I have no custom css applying.

![image](https://user-images.githubusercontent.com/98420603/153421207-632b33e7-e0d2-484a-9317-bfee66270624.png)
![image](https://user-images.githubusercontent.com/98420603/153421394-6c043259-80bc-416e-a369-66a0bf7723ed.png)
